qiskit/optimization/problems/constraint.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Abstract Constraint."""
+
+from abc import abstractmethod
+from enum import Enum
+from typing import Union, List, Dict
+
+from numpy import ndarray
+
+from qiskit.optimization import QiskitOptimizationError
+from qiskit.optimization.problems.has_quadratic_program import HasQuadraticProgram
```

Comment:
Can we use relative imports in these modules in problems where they import a peer module instead of fully qualified